apps/emqx/src/emqx_schema_secret.erl

```diff
@@ -71,8 +71,8 @@ convert_secret(Secret, #{}) ->
     end.
 
 -spec wrap(source()) -> emqx_secret:t(t()).
-wrap(<<"file://", Filename/binary>>) ->
-    emqx_secret:wrap_load({file, Filename});
+wrap(<<"file://", _Filename/binary>> = Secret) ->
+    emqx_secret:wrap_load({file, Secret});
```

Comment:
Q: Why this change is needed?

Reply:
this is to keep the original prefix so there is no need to concatenate "file://" after calling `emqx_secret:term`.

Reply:
Just to clarify: the idea was to keep this `file://...` convention on the schema level only, so that lower-level code would not have to know about it.